infra-as-code/terraform/modules/studio/main.tf

```diff
@@ -0,0 +1,36 @@
+terraform {
+  required_providers {
+    twilio = {
+      source  = "twilio/twilio"
+      version = ">=0.4.0"
+    }
+  }
+}
+
+resource "twilio_studio_flows_v2" "chat" {
+  friendly_name  = "Chat Flow"
+  status         = "published"
+  definition = templatefile("../../studio/chat-flow.json", local.params)
+}
+
+resource "twilio_studio_flows_v2" "messaging" {
+  friendly_name  = "Messaging Flow"
+  status         = "published"
+  definition = templatefile("../../studio/messaging-flow.json", local.params)
+}
+
+resource "twilio_studio_flows_v2" "voice" {
+  friendly_name  = "Voice IVR"
+  status         = "published"
+  definition = templatefile("../../studio/voice-flow.json", local.params)
+}
```

Comment:
Maybe it is worth adding "Template Example" to these Studio flow names as well?

Reply:
Im not a fan of that, i like that everything works from a vanilla account after a deploy - if we cant target these it undermines some of the value.

there is an unmissable call out that this will overwrite your config in the setup instructions, i think that is sufficient.